### PR TITLE
Added a dev doc for mockups with Balsamiq

### PR DESCRIPTION
**Pull request by SilkeSchomann**
**Description of work.**

Added a page to the developer documentations with a brief introduction to mock-ups with Balsamiq Wireframes

**To test:**

Please read the Balsamiq page and check spelling, grammar and formatting.

*There is no associated issue.*

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: dev-docs/source/BalsamiqWireframes.rst [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+We have licenses for the cloud version of Balsamiq: https://balsamiq.cloud/
+Please use your STFC email address to register and access the STFC space:
+
+.. image:: images/balsamiq_cloud.png
+
+Create a project by clicking the ``+`` (plus) in the upper left hand corner:
+
+.. image:: images/balsamiq_new_project.png
+
+The new project automatically the editor with the canvas and the UI library as well as several toolbars and panels:
```
Comment (robertapplin) [inline dev-docs/source/BalsamiqWireframes.rst:35]: I think there's a typo in this sentence.

**Review by robertapplin — CHANGES_REQUESTED** [review]
Balsamiq mockups are useful so its good to have some documentation about them. Just one typo I think

File: dev-docs/source/BalsamiqWireframes.rst (hunk 1 above)
Comment (SilkeSchomann) [reply dev-docs/source/BalsamiqWireframes.rst]: Good spot! I've submitted a correction.

**Review by robertapplin — APPROVED** [review]
Looks good to me

**Pull request merged**